internal/dag/dag.go

```diff
@@ -335,6 +335,9 @@ type Route struct {
 	// RateLimitPolicy defines if/how requests for the route are rate limited.
 	RateLimitPolicy *RateLimitPolicy
 
+	// RateLimitPerRoute defines how the route should handle rate limits defined by the virtual host.
+	RateLimitPerRoute *RateLimitPerRoute
```

Comment:
I think we should be able to use reuse  `RateLimitPolicy.disabled`. that will allow us to not introduce new fields.

Reply:
So `dag. RateLimitPolicy` doesn't really have `disabled` on it, this noob is only on the API.

I don't believe introducing this switch to DAG is a good idea as DAG is closer to Envoy than to the HTTPProxy API and also `RateLimitPerRoute` is native to Envoy's struct for doing exactly this.